pkgs/applications/editors/vim/macvim-configurable.nix

```diff
@@ -29,7 +29,7 @@ let
           "/Applications/MacVim.app/Contents/MacOS"
           "/Applications/MacVim.app/Contents/bin"
         ];
-        buildInputs = [ makeWrapper ];
+        nativeBuildInputs = [ makeWrapper ];
```

Comment:
This is a `buildEnv` (which doesn't support `nativeBuildInputs`). I can submit a PR to revert the change in this file, but I wonder if other packages were broken this way as well, and whether `buildEnv` should perhaps support `nativeBuildInputs` after all?